projects/packages/waf/src/class-waf-runner.php

```diff
@@ -65,6 +65,19 @@ public static function is_allowed_mode( $option ) {
 		return in_array( $option, $allowed_modes, true );
 	}
 
+	/**
+	 * Determines if the WAF module is enabled on the site.
+	 *
+	 * @return bool
+	 */
+	public static function is_enabled() {
+		if ( class_exists( 'Jetpack' ) ) {
+			return \Jetpack::is_module_active( 'waf' );
```

Comment:
This `Jetpack` plugin method is only a wrapper for the `Modules()->is_active()` [package method](https://github.com/Automattic/jetpack/blob/master/projects/packages/status/src/class-modules.php#L25). Does it make sense to use that now to avoid refactoring when we bring it into the Protect plugin (assuming we are)?

Reply:
Removed the usage of the `Jetpack` method via https://github.com/Automattic/jetpack/pull/23769/commits/5b4c7a07b1970d2da98c4b0d61ba047b60a3d717 :+1: